composer/loggers/object_store_logger.py

```diff
@@ -294,6 +294,20 @@ def log_file_artifact(self, state: State, log_level: LogLevel, artifact_name: st
         object_name = self._object_name(artifact_name)
         self._file_upload_queue.put_nowait((copied_path, object_name, overwrite))
 
+    def log_symlink_artifact(self, state: State, log_level: LogLevel, existing_artifact_name: str,
+                             symlink_artifact_name: str, overwrite: bool):
+        # Only symlink if we're logging artifact to begin with
```

Comment:
Could you add a bit of a description here that object stores do not natively support symlinks, so we emulate symlinks with a `.symlink` file, where this file contains the object name that it points to?